web/themes/new_weather_theme/assets/sass/components/_radar.scss

```diff
@@ -0,0 +1,11 @@
+@use "uswds-core" as *;
+
+.wx-radar-container {
+  aspect-ratio: 1/1;
+}
+
+@include at-media-max("tablet") {
```

Comment:
`at-media-max` is cool to know about! 

**non-blocking:** I am totally fine with leaving it this way, but the thought occurred to me that for other components we style mobile-first, and do the media query for the larger sizes.

Reply:
I'd need to check if there's an equivalent `at-media-min` in order to do that, but good call! I'll take a look.

Reply:
Oh, `at-media` is the min version. Ha. Update incoming.

Reply:
@igorkorenfeld Would you ask @partly-igor to take a look when he's got time? :trollface: